build/depvers.sh

```diff
@@ -32,33 +32,36 @@ pkginfo=($(go list -f '{{.Dir}} {{.Root}}' ${pkgs} 2>/dev/null))
 # Loop over the package info which comes in pairs in the pkginfo
 # array.
 for (( i=0; i < ${#pkginfo[@]}; i+=2 )); do
-    dir=${pkginfo[$i]}
-    if ! test -d "${dir}"; then
-	continue
-    fi
+  dir=${pkginfo[$i]}
+  if ! test -d "${dir}"; then
+  continue
+  fi
 
-    toplevel=$(git -C "${dir}" rev-parse --show-toplevel 2>/dev/null)
+  toplevel=$(git -C "${dir}" rev-parse --show-toplevel 2>/dev/null)
 
-    git=1
+  git=1
+  if test "${toplevel}" = ""; then
+    toplevel=$(hg --cwd "${dir}" root 2>/dev/null)
     if test "${toplevel}" = ""; then
-	toplevel=$(hg --cwd "${dir}" root 2>/dev/null)
-	if test "${toplevel}" = ""; then
-	    # TODO(pmattis): Handle subversion/bazaar.
-	    continue
-	fi
-	git=0
+      # TODO(pmattis): Handle subversion/bazaar.
+      continue
     fi
+    git=0
+  fi
 
-    if visit "${toplevel}"; then
-	continue
-    fi
+  if visit "${toplevel}"; then
+    continue
+  fi
 
-    if test "${git}" -eq 1; then
-	vers=$(git -C "${dir}" rev-parse HEAD 2>/dev/null )
-    else
-	vers=$(hg --cwd "${dir}" parent --template '{node}')
-    fi
+  if test "${git}" -eq 1; then
+    vers=$(git -C "${dir}" rev-parse HEAD 2>/dev/null )
+  else
+    # file ownership sometimes doesn't work in docker and hg gets grumpy about
+    # this file, which is basically empty anyway, so just delete it
+    rm "${dir}/.hg/hgrc"
+    vers=$(hg --cwd "${dir}" parent --template '{node}')
+  fi
 
-    root=${pkginfo[$i+1]}
-    echo ${toplevel#$root/src/}:${vers}
+  root=${pkginfo[$i+1]}
+  echo ${toplevel#$root/src/}:${vers}
```

Comment:
Ok. I propose you pull this out into a separate change then and normalize the whitespace in all of the .sh scripts. For example, `deplicense.sh` could use the same treatment.